src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep.java

```diff
@@ -92,6 +94,8 @@
                 MultiBinding.MultiEnvironment environment = binding.bind(run, workspace, launcher, listener);
                 unbinders.add(environment.getUnbinder());
                 overrides.putAll(environment.getValues());
+                IdCredentials id = CredentialsProvider.findCredentialById(binding.getCredentialsId(), binding.type(), run);
+                CredentialsProvider.track(run, id);
```

Comment:
Perhaps also desirable to call `track(Node, Credentials)`. (For `BindingStep`, just inject a `Node`; for `SecretBuildWrapper`, use the method on `AbstractBuild` IIRC.) @stephenc any guidance?